sources/packages/backend/libs/sims-db/src/entities/application-student-file.model.ts

```diff
@@ -31,9 +25,6 @@ export class ApplicationStudentFile extends RecordDataModel {
   })
   application: Application;
 
-  @RelationId((file: ApplicationStudentFile) => file.studentFile)
```

Comment:
@guru-aot Just to confirm, the `@RelationId`(s) that you have removed as a part of this PR aren't used in our queries and the ones for which you have created the story are the ones that are being used (tied up in some way) and therefore need to be removed later?

Also, can you please briefly tell how this was putting the db in recovery mode.

Reply:
As discussed in the meeting, i believe you might have got the reason.